config/samples/awscloudwatchlog-sinkbinding.yaml

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2020 TriggerMesh Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Sample ContainerSource for the AWS CodeCommit event source.
+
+apiVersion: sources.knative.dev/v1alpha2
+kind: ContainerSource
+metadata:
+  name: &srcname awscloudwatchlog-source
+spec:
+  template:
+    spec:
+      containers:
+        - name: source
+          image: gcr.io/triggermesh/awscloudwatchlogsource:latest
+
+          env:
+
+            # LogWatch ARN with LogGroup
+            - name: ARN
+              value: 'arn:aws:logs:us-west-2:043455440429:log-group:/aws/lambda/lambdadumper:*'
```

Comment:
It will. The `:*` would normally denote the `logStream` where the actual log entries will be streamed to.  The `*` catch-all, as referenced by the AWS console, will return all streams. The source adapter relies on this to iterate through all streams to find which ones have entries made since the last polled interval, and send them as cloud events.